app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt

```diff
@@ -1,20 +1,90 @@
 package org.feature.fox.coffee_counter.ui
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Divider
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.feature.fox.coffee_counter.R
+import org.feature.fox.coffee_counter.data.local.database.tables.User
 
 @Preview(showSystemUi = true)
 @Composable
 fun UsersView() {
-    Box(
-        modifier = Modifier.fillMaxSize(),
-        contentAlignment = Alignment.Center
-    ){
-        Text(text = "Users screen")
+    val users = listOf(
+        User(id = "a", name = "Julian", isAdmin = true, password = "julian"),
+        User(id = "b", name = "Steffen", isAdmin = true, password = "steffen"),
+        User(id = "c", name = "Kevin", isAdmin = true, password = "kevin"),
+        User(id = "d", name = "Nils", isAdmin = false, password = "nils"),
+    )
+    Column {
+        SearchBar()
+        UserList(users)
     }
 }
+
+@Composable
+fun UserList(users: List<User>) {
+    Column {
+        Text(
+            stringResource(id = R.string.user_list_title),
+            fontWeight = FontWeight.Medium,
+            fontSize = 30.sp,
+            modifier = Modifier.padding(start = 5.dp)
+        )
+        Column(
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(5.dp)
+                .fillMaxWidth()
+                .weight(1f),
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            users.forEach { user ->
+                UserRow(user)
+                Divider(
+                    color = Color.Gray,
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    thickness = 1.dp
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun UserRow(user: User) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        // Not sure if there's a better inline solution
```

Comment:
You can also remove this comment, but i will already give you my approve.

Reply:
Removed it